ramp-database/rampdb/tools/__init__.py

```diff
@@ -2,60 +2,3 @@
 The :mod:`rampdb.tools` provides some routine to get and set information from
 and in the database.
 """
-
```

Comment:
I thought that it would be more readable to import stuff as:

```python

from rampdb.model.submission import SomeModel

from rampdb.tools.submission import get_function_*

```

It is a bit more longuish but a bit more clear what we are importing (either tools or models).